nexus/db-queries/src/authz/api_resources.rs

```diff
@@ -498,6 +498,66 @@ impl AuthorizedResource for DeviceAuthRequestList {
     }
 }
 
+/// Synthetic resource describing the list of Certificates associated with a
+/// Silo
+#[derive(Clone, Debug, Eq, PartialEq)]
+pub struct SiloCertificateList(Silo);
```

Comment:
I know we've talked about this a little bit, but I just want to clarify... is the reason this needs to have a synthetic resource because it's permissions differ in some meaningful way from other silo resources or is it because you think all silo resources (or every resource) should have a synthetic resource?

Reply:
The obvious alternative would be directly checking "list children" on the Silo.  Compared to that, a synthetic resource gives us two things: first what you said that the policy is different for certificates than for, say, projects (both of which are Silo children); and second: the policy ("a person can list the certificates in the Silo if they are an administrator of the Silo's parent fleet") can be separated from the code that checks it (which just asks "can this person list the certificates in the Silo").  This makes the code easier to write and verify (since you're not making any judgments about the policy there) and the policy is easier to write and verify too (since you're not worrying about implementation details there).